docs/content/docs/try-table-store/quick-start.md

```diff
@@ -48,24 +48,7 @@ tar -xzf flink-*.tgz
 Download [flink-table-store-dist-{{< version >}}.jar](https://repo.maven.apache.org/maven2/org/apache/flink/flink-table-store-dist/{{< version >}}/flink-table-store-dist-{{< version >}}.jar).
 {{< /stable >}}
 {{< unstable >}}
-
-You are using an unreleased version of Table Store, in order to build Table Store
-bundle jar you need the source code.
-
-To clone from git, enter:
-
-```bash
-git clone {{< github_repo >}}
-```
-
-The simplest way of building Table Store is by running:
-
-```bash
-mvn clean install -DskipTests
-```
-
-You can find the bundle jar in `"./flink-table-store-dist/target/flink-table-store-dist-{{< version >}}.jar"`.
-
+You are using an unreleased version of Table Store, you need to manually [Build Bundled Jar]({{< ref "docs/engines/build" >}}) from the source code.
```

Comment:
```suggestion
If you are using an unreleased version of Table Store, you need to manually [Build Bundled Jar]({{< ref "docs/engines/build" >}}) from the source code.
```

Reply:
`{{< unstable >}}`, it is a unreleased version.